client/blocks/reader-subscription-list-item/email-settings.jsx

```diff
@@ -106,5 +151,29 @@ class ReaderEmailSubscriptionSettingsPopout extends Component {
 	}
 }
 
-export default localize( ReaderEmailSubscriptionSettingsPopout );
+const mapStateToProps = ( state, ownProps ) => {
+	const follow = find( getReaderFollows( state ), { blog_ID: ownProps.siteId } );
+
+	const currentDeliveryMethods = get( follow, [ 'delivery_methods', 'email' ], {} );
```

Comment:
i wonder if this would be clearer or worse with destructuring

```
const { 
  delivery_methods: { 
    email: { 
      send_posts: notifyOnNewPosts = false,
      post_delivery_frequency: deliveryFrequency,
      send_comments: notifyOnNewComments = false 
} = follow;